app/src/components/UI/Pages/Login.tsx

```diff
@@ -137,4 +150,17 @@ export default function Login(): React.ReactElement {
       </Formik>
     </>
   );
-}
+};
+
+const LoginWrapper: React.FC = () => {
+  useGetTokenQuery();
+  const token = useAppSelector((state) => state.token.token);
+
+  if (token === null) {
```

Comment:
If this is shown instead of the Login component if I am not logged in and token is null, how can I log in?